src/run.ts

```diff
@@ -43,6 +43,14 @@ class HelmRenderEngine extends RenderEngine {
         console.log("Getting helm verion..");
         let isV3 = true;
         await this.isHelmV3(helmPath).then(() => { isV3 = true }).catch(() => { isV3 = false });
+        
+        try {
+            if (!isV3) {
+                await utilities.execCommand(helmPath, ['init', '--client-only'], options);
+            }
+        } catch (ex) {
+            core.warning(util.format('Could not run helm init command: ', ex));
```

Comment:
This case shouldn't ever hit because of the `v3` check.

Reply:
The check is for the cases when version is NOT `v3` and helm init command is not supported. I added the catch block just to be on the safe side.

Reply:
Seems redundant. But it's fine :)